source/core_sntp_client.c

```diff
@@ -698,9 +713,11 @@ static SntpStatus_t processServerResponse( SntpContext_t * pContext,
         {
             /* Server has responded successfully with time, and we have calculated the clock offset
              * of system clock relative to the server.*/
-            LogDebug( ( "Updating system time: ServerTime=%u %ums ClockOffset=%lums",
-                        parsedResponse.serverTime.seconds, FRACTIONS_TO_MS( parsedResponse.serverTime.fractions ),
-                        parsedResponse.clockOffsetMs ) );
+            LogDebug( ( "Updating system time: ServerTime=%lu %lums ClockOffset=%lds",
+                        ( unsigned long ) parsedResponse.serverTime.seconds,
+                        ( unsigned long ) FRACTIONS_TO_MS( parsedResponse.serverTime.fractions ),
+                        /* Print out in seconds instead of Ms to account for C90 lack of %lld */
+                        ( long ) parsedResponse.clockOffsetMs / 1000 ) );
```

Comment:
`clockOffsetMs` is a `int64_t` as seen [here](https://github.com/FreeRTOS/coreSNTP/blob/v1.2.0/source/include/core_sntp_serializer.h#L345), this fixes the issue of this signed number being incorrectly printed as an unsigned number.